tests/llm/utils/mock_toolset.py

```
@@ -440,37 +464,67 @@ def _invoke(
-                logging.debug(str(e))
-
-            result = self._call_live_invoke(params)
-            # Write mock file
-            mock_file_path = self._file_manager.write_mock(
-                tool_name=self.name,
-                toolset_name=self._toolset_name,
-                params=params,
-                result=result,
-            )
+            else:
+                raise ValueError(f"Unknown mock mode: {self._mode}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)